packages/makeup-typeahead/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "makeup-typeahead",
+  "description": "Captures letter keys pressed and returns a list of relevant strings",
+  "version": "0.1.0",
```

Comment:
Nitpick, but can you drop this to 0.0.1? After that I think we are good to publish to NPM!

Reply:
done!